client/src/components/Metadata.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import Options from "./Options.jsx";
+import Languages from "./Languages.jsx";
+import Rating from "./Rating.jsx";
+import InfoPanel from "./InfoPanel.jsx";
+import Links from "./Links.jsx";
+import { Container } from './styles/styles'
+
+class Metadata extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
+            isLoaded: false,
+            gameId: null,
+            options: [],
+            languages: [],
+            rating: null,
+            infoPanel: []
+        }
+    }
+
+    getMetadata() {
+
+    }
+
+    componentDidMount() {
+        fetch("/metadata")
+            .then((res) => {
+                return res.json()
+            })
+            .then(
+                (result) => {
+                    this.setState({
+                        gameId: result[0],
+                        options: result[1],
+                        languages: result[2],
+                        rating: result[3],
+                        infoPanel: result[4],
+                        isLoaded: true
+                    });
+                },
+                // Note: it's important to handle errors here
+                // instead of a catch() block so that we don't swallow
+                // exceptions from actual bugs in components.
+                (error) => {
```

Comment:
this is an interesting technique to not use the catch block, is this fetch specific or is this a technique you can use on any promise chain?